### PR TITLE
feat: add option to hide FPS counter

### LINKED ISSUE #1473: Ability to turn off "FPS #" overlay on camera dashboard
### Requested feature:

Especially when showing multiple cameras on dashboard, and on mobile the "FPS #" overlay makes it hard for me to see some details. Although useful when setting things up, I don't need to see the current FPS always. And wish there was a way to just turn this off. 

### Solves the following problem:

_No response_

### Additional information:

_No response_

### PR DESCRIPTION
**Pull request by meteyou**
Signed-off-by: Stefan Dej <meteyou@gmail.com>

## Description

This PR add an option for each mjpegstreamer webcam to hide the FPS counter.

## Related Tickets & Documents

fixes #1473

## Mobile & Desktop Screenshots/Recordings

image

## [optional] Are there any post-deployment tasks we need to perform?

none

### CODE REVIEW

File: src/components/webcams/Mjpegstreamer.vue [hunk 1]
```diff
@@ -73,6 +73,12 @@ export default class Mjpegstreamer extends Mixins(BaseMixin, WebcamMixin) {
         return this.currentFPS < 10 ? '0' + this.currentFPS.toString() : this.currentFPS
     }
 
+    get showFpsCounter() {
+        if (!this.showFps) return false
+
+        return this.camSettings.extra_data?.hideFps ?? true
```
Comment (rackrick) [inline src/components/webcams/Mjpegstreamer.vue:79]: invert bool. setting is hide fps, function is show
```suggestion
        return !this.camSettings.extra_data?.hideFps ?? true
```

Comment (meteyou) [reply src/components/webcams/Mjpegstreamer.vue]: fixed with the new commit

**Review by rackrick — APPROVED** [review]

**Comment by meteyou** [comment]
> is there any way to do this for webcams configured in moonraker.conf?

no sry, moonraker doesnt have this option. do you need this for a "default setup"?

**Comment by meteyou** [comment]
mainsail use the moonraker webcam api to config the webcam. so if you just setup the webcam in mainsail, you should also have it identical in mobilracker. i dont know how you read this settings in home assistant, but if you also use the moonraker webcam api, its just identical. the hideFps field is just a value in the extra directory.

**Pull request merged**